tesseract_core/sdk/engine.py

```diff
@@ -113,6 +113,10 @@ def wrapper_needs_docker(*args: Any, **kwargs: Any) -> None:
             raise UserError(
                 "Could not reach Docker daemon, check if it is running."
             ) from ex
+        except FileNotFoundError as ex:
+            raise UserError(
+                "Docker daemon is not running, check if it is installed."
```

Comment:
Maybe a better wording, as this should be triggered by the executable not being found

```suggestion
                "Docker not found, check if it is installed."
```

...but also, this makes me think; should we eliminate references to docker in error messages, now that we support podman and so on? on the one hand, it would not mislead users in thinking that they have to install docker, as a "check if docker is installed" message; on the other a message like "containerization engine missing" is probably less clear. Wdyt @dionhaefner @apaleyes ?

Reply:
Also, I just saw after writing the above that a similar strategy was suggested by @heitorPB as well [here](https://github.com/pasteurlabs/tesseract-core/pull/216), and I really like his solution of dynamically inserting the name

Reply:
The thing is his error was actually slightly different (I originally thought it was a dupe but it was not), and I'm not sure we would be able to know what the executable is called in this workflow as we are just calling `docker info` to see if it's available and if not, then we would get the error as we don't know what underlying containerization engine we are using... Whereas in the other one it is part of the update config workflow so it has access to that information (?)